ext/opentelemetry-ext-azure-monitor/src/opentelemetry/ext/azure_monitor/metrics.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from opentelemetry.ext.azure_monitor import protocol, transport, util
+from opentelemetry.sdk.metrics.export import (
+    MetricsExporter,
+    MetricsExportResult,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class AzureMonitorMetricsExporter(MetricsExporter, transport.TransportMixin):
+    def __init__(self, **options):
+        self.options = util.Options(**options)
+        util.validate_key(self.options.instrumentation_key)
+        self.export_result_type = MetricsExportResult
+
+    def export(self, metric_tuples):
+        # Metric tuples is a sequence of metric to label values pairs
```

Comment:
tuples -> tuple.

values -> value.

Reply:
I am describing the parameter.
Changed the comment to:
`# metric_tuples is a sequence of metric to label value pairs`

Reply:
In this case, should we use `"""` comments?